src/backend/app/Console/Commands/DemoDataCreator.php

```diff
@@ -265,6 +267,22 @@ private function generateTransaction(): void {
             $token->save();
             $transactionData->token = $token;
 
+            // generate meter_token
+            $meterTokenData = [
+                'meter_id' => $randomMeter->id,
+                'token' => Str::random(30),
+                'energy' => round(
+                    $transactionData->transaction->amount /
+                    $randomMeter['tariff']['price'],
+                    2
+                ),
+                'transaction_id' => $transaction->id,
+            ];
+            $meterToken = $this->meterToken->newQuery()->make(['meter_id' => $meterTokenData['meter_id'],
+                'token' => $meterTokenData['token'], '' => $meterTokenData['energy'],
+                'transaction_id' => $meterTokenData['transaction_id']]);
+            $meterToken->save();
```

Comment:
Did you manage to understand what is the difference between `meterToken` and the `token` from above?

Reply:
Basically, I'm looking at this change: https://github.com/EnAccess/micropowermanager/commit/c85f9aa0c6cdedb468dbe91a96f2c89448faa853#diff-588c01299a5ad9020a72f27600dc3dfd9cc8c4fbd8afaeb2917d0e9090f71d6d

What I'm wondering is: Is `MeterToken` deprecated? I.e. was `MeterToken` replaced by the more generic `Token` model?

Reply:
That sounds more plausible since the frontend does expect to get tokens from transactions. 

Alright, I will go with this assumption.